shared-bitrise.yml

```diff
@@ -32,6 +38,12 @@ workflows:
 
             bundle exec pod spec lint --allow-warnings
             bundle exec pod trunk push --allow-warnings --verbose
+    - deploy-to-bitrise-io@2:
+        is_skippable: true
+        inputs:
+        - is_enable_public_page: "false"
+        - deploy_path: ./artifacts
+        - is_compress: "true"
```

Comment:
nit: seems to be duplicate code. Can it go in a shared "thing" somehow? (np if not)

Reply:
Ok. I moved those steps to a new `_deploy_artifacts` workflow